cmd/plugin-manager/add/add.go

```diff
@@ -46,7 +46,7 @@ func (o *Options) Validate(args []string) error {
 	if len(args) != 1 {
 		return errors.New("please input only one plugin name")
 	}
-
+	o.PluginDir = fmt.Sprintf("%v/%v", o.PluginDir, plugin.MANAGED_DIR)
```

Comment:
This seems risky. If  Validate ever got called twice, o.PluginDir might end up being "plugins/managed/managed" -- I think o.PluginDir should probably stay as-is but maybe define a func:
```
func (o *Options) ManagedPluginDir() string {
    return fmt.Sprintf("%v/%v", o.PluginDir, plugin.MANAGED_DIR)
}
```

Reply:
Then use `o.ManagedPluginDir()` instead of `o.PluginDir` any time you need the actual directory where managed plugins go.